routes/api/contacts.js

```diff
@@ -46,8 +46,8 @@ router.post("/api/contacts", async (req, res, next) => {
 
 router.delete("/api/contacts/:id", async (req, res, next) => {
   try {
-    await contacts.removeContact(req.params.id);
-    res.json({ message: "Contact deleted" });
+    const deleteContact = await contacts.removeContact(req.params.id);
+    res.json(deleteContact);
```

Comment:
Tu może zostaw że kontakt usunięto. Bo jeżeli w `contacts.removeContact` zmienisz tak że tam zrobisz też throw error w bloku catch to tutaj nigdy nic nie zostanie zwrócone, bo tam nie ma return już.